### ISSUE TITLE
The logo should have alt text for accessibility

### ISSUE THREAD

**Issue by carolyncole** [pulibrary/orangelight#2236]
The logo img tag should have `alt="Link to Princeton University Library home page"` . It fails a lighthouse audit for both requests and orangelight.
![Screen Shot 2020-09-04 at 3 01 46 PM](https://user-images.githubusercontent.com/1599081/92276728-925db380-eebf-11ea-9498-90a0f4f104d5.png)
